Makefile

```
@@ -115,7 +124,8 @@ configure: configure.ac aclocal.m4 build-aux/ocaml_version.m4 tools/autogen
 
 .PHONY: partialclean
 partialclean::
-	rm -f utils/config.ml utils/domainstate.ml utils/domainstate.mli
+	rm -f utils/config.ml utils/config_boot.ml* utils/config_main.ml* \
```

We don't use the `ml*` wildcard anywhere else in the makefiles, probably for good reason (it looks dangerous to me at least).
```suggestion
	rm -f utils/config.ml \
	      utils/config_boot.ml utils/config_boot.mli \
	      utils/config_main.ml utils/config_main.mli \
```

---

That’s a very good point, thanks!

---

And it now pushed